engines/auth_by/app/models/auth_by/user.rb

```diff
@@ -1,11 +1,9 @@
 # frozen_string_literal: true
 
 module AuthBy
-  class User < CoreBy::User
+  class User < ::CoreBy::Base::ApplicationRecord
```

Comment:
Maybe, we should use `Engine.config.user_class.constantize` here as well?

And also, check for the presence of the required attributes?

Reply:
Or, another idea, is to use some kind of delegation and put all the required authentication logic into a separate model/table with the link to a real User? (`has_one :authenticateble, class_name: Engine.config.auth_class_name`?)